packs/data/spells.db/protection.json

```diff
@@ -26,14 +27,19 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You ward a creature against a specified alignment. Choose chaotic, evil, good, or lawful when you cast this spell. The target gains a +1 status bonus to Armor Class and saving throws against creatures and effects of the chosen alignment. This bonus increases to +3 against effects from such creatures that would directly control the target and against attacks made by summoned creatures of the chosen alignment.</p>\n<p>This spell gains the trait that opposes the alignment you chose-if you choose chaos, this spell gains the lawful trait, and vice versa; if you choose evil, this spell gains the good trait, and vice versa.</p>\n<p>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Chaotic]{Spell Effect: Protection from Chaotic}</p>\n<p>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Evil]{Spell Effect: Protection from Evil}</p>\n<p>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Good]{Spell Effect: Protection from Good}</p>\n<p>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Lawful]{Spell Effect: Protection from Lawful}</p>"
+            "value": "<p>You ward a creature against a specified alignment. Choose chaotic, evil, good, or lawful when you cast this spell. The target gains a +1 status bonus to Armor Class and saving throws against creatures and effects of the chosen alignment. This bonus increases to +3 against effects from such creatures that would directly control the target and against attacks made by summoned creatures of the chosen alignment.</p>\n<p>This spell gains the trait that opposes the alignment you chose-if you choose chaos, this spell gains the lawful trait, and vice versa; if you choose evil, this spell gains the good trait, and vice versa.</p>\n<hr />\n<ul>\n<li>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Chaotic]{Spell Effect: Protection from Chaotic}</li>\n<li>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Evil]{Spell Effect: Protection from Evil}</li>\n<li>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Good]{Spell Effect: Protection from Good}</li>\n<li>@Compendium[pf2e.spell-effects.Spell Effect: Protection from Lawful]{Spell Effect: Protection from Lawful}</li>\n</ul>"
         },
         "duration": {
             "value": "1 minute"
         },
         "hasCounteractCheck": {
             "value": false
         },
+        "heightening": {
+            "damage": {},
+            "interval": 1,
+            "type": "interval"
+        },
```

Comment:
Why?

Reply:
On Protection, all I edited was the description text to add the `<hr />` tag and change the paragraphs that had the Spell Effects into an unordered list. This change happened from opening the spell in this version of the system.

Reply:
New commit removes that; I'm not certain why it was added, because I did not click on either of the Heightening buttons on the spell.